docs/docs/account/user-permissions.mdx

```diff
@@ -0,0 +1,61 @@
+|       Tags        |        No         |              No               |         View<br></br>Add<br></br>Edit         |                         View                          |             View<br></br>Add<br></br>Edit             |             View<br></br>Add<br></br>Edit             |
+| Slack Integration |        No         |              No               |         View<br></br>Add<br></br>Edit         |                         View                          |             View<br></br>Add<br></br>Edit             |             View<br></br>Add<br></br>Edit             |
+|  Manage Projects  |        No         |              No               |                      No                       |                          No                           |                          No                           |                          Yes                          |
+|    Manage Team    |        No         |              No               |                      No                       |                          No                           |                          No                           |                          Yes                          |
+|    Manage Plan    |        No         |              No               |                      No                       |                          No                           |                          No                           |                          Yes                          |
+|  Manage Billing   |        No         |              No               |                      No                       |                          No                           |                          No                           |                          Yes                          |
+
+#### Environment Specific Limits
+
+![Environment Specific Limits](/images/user-permissions-env-specific.png)
+
+And finally, there is one last level to GrowthBook's user permissions, which determines environment specific limits. This permission level only applies to `engineer` and `experimenter` roles in organizations in a `Pro` or `Enterprise` account. This
+allows you to limit the feature flags and experiments a user can manage to particular environments. For instance, you can allow and engineer to create and run experiments in a `staging` environment, but not in `production`.
+
+#### How permissions are evaluated
+
+When evaluating whether or not a user has permissions to perform a certain action, GrowthBook will check if the action is taking place within a particular project, and if so, we'll check their project-level permissions first, before falling back to
+the user's global role. If the action is not taking place within a project, we'll just check the user's global role.
+
+If your organization is not enrolled in a `Pro` or `Enterprise` account, we will simply evaluate the user's permission level via their global role.
```

Comment:
And some more chat-gpt inspired rephrasings that I think are slightly more clear here:

GrowthBook's user permissions also include environment specific limits. This permission level applies only to `engineer` and `experimenter` roles in `Pro` or `Enterprise` accounts. It allows you to limit the feature flags and experiments a user can manage to specific environments. For example, you can allow an engineer to create and run experiments in a `staging` environment, but not in `production`.

#### How permissions are evaluated

GrowthBook evaluates user permissions based on whether an action is taking place within a specific project. If so, project-level permissions are checked first, followed by the user's global role. If the action is not within a project, only the user's global role is checked.

For organizations without a `Pro` or `Enterprise` account, user permissions are evaluated solely based on their global role.